api/lib/application/competence-evaluations/competence-evaluation-controller.js

```diff
@@ -5,14 +5,14 @@ const { BadRequestError } = require('../../infrastructure/errors');
 
 module.exports = {
 
-  start(request, h) {
+  async startOrResume(request, h) {
     const userId = request.auth.credentials.userId;
-    const competenceId = request.payload.data.attributes['competence-id'];
-    return usecases.startOrResumeCompetenceEvaluation({ competenceId, userId })
-      .then(({ created, competenceEvaluation }) => {
-        const serialized = serializer.serialize(competenceEvaluation);
-        return created ? h.response(serialized).created() : serialized;
-      });
+    const competenceId = request.query.competenceId;
```

Comment:
Quelle est la différence entre une payload et une query dans le cadre d'un POST ? 
Une query me semble associée à une requête GET / FIND.
Une payload me semble associée à un PATCH / POST.
Est-ce que ce n'est pas contre-intuitif ?

Reply:
Dans certains cas on peut avoir des queryParams dans un GET. Je t'invite à regarder cette discussion pour rentrer plus dans le détails :
https://stackoverflow.com/questions/611906/http-post-with-url-query-parameters-good-idea-or-not

Reply:
OK techniquement il n'y a rien qui interdit d'avoir une querystring sur un POST, mais il me semble qu'on ne l'a pas fait jusqu'ici dans Pix, et je ne vois pas de raison de changer pour ce cas-ci ?

Reply:
J'entend la phrase `il me semble qu'on ne l'a pas fait jusqu'ici dans Pix`, et par soucis de cohérence dans le code, la solution d'utiliser le payload me parait plus sain.